AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -600,7 +615,7 @@ - (void)rangeController:(ASRangeController *)rangeController didInsertNodesAtInd
   }
 }
 
-- (void)rangeController:(ASRangeController *)rangeController didDeleteNodesAtIndexPaths:(NSArray *)indexPaths withAnimationOptions:(ASDataControllerAnimationOptions)animationOptions
+- (void)rangeController:(ASRangeController *)rangeController didDeleteNodes:(NSArray *)nodes atIndexPaths:(NSArray *)indexPaths withAnimationOptions:(ASDataControllerAnimationOptions)animationOptions
```

Comment:
I appreciate you explicitly calling out the performance cost of adding this.  Because I have plans to do a detailed and focused optimization effort for these code paths in the next few months (probably ASDK 2.1, Oct or Nov), I'm not concerned about this.  The impact is likely small for most use cases, although it is also possible to construct cases where the overhead could be significant.  The feature is worth the anticipated cost in this case.